File: .sdk/tm/rb/utility/make_context.rb
# PassantenfrequenzStadtStgallen SDK utility: make_context
require_relative '../core/context'
module PassantenfrequenzStadtStgallenUtilities
  MakeContext = ->(ctxmap, basectx) {
    PassantenfrequenzStadtStgallenContext.new(ctxmap, basectx)
  }
end

File: .sdk/tm/py/pkg/utility/make_context.py
# PassantenfrequenzStadtStgallen SDK utility: make_context

from projectname_sdk.core.context import PassantenfrequenzStadtStgallenContext


def make_context_util(ctxmap, basectx):
    return PassantenfrequenzStadtStgallenContext(ctxmap, basectx)
